scripts/revng_translate/translate.py

```diff
@@ -0,0 +1,110 @@
+                        type=str,
+                        nargs=1,
+                        help="Input binary to be translated")
+
+    parser.add_argument("--show-command",
+                        dest="showCommand",
+                        action="store_true",
+                        help="show the auto-Pipe command and exit")
+
+    parser.add_argument("-o",
+                        "--output",
+                        metavar="OUTPUT",
+                        dest="output",
+                        type=str,
+                        nargs="?",
+                        help="Translated output")
+
+    parser.add_argument("-v",
+                        "--verbose",
+                        action='store_true',
+                        dest="verbose",
+                        help="verbose")
+
+    parser.add_argument("-i",
+                        "--isolate",
+                        action="store_true",
+                        help="Enable function isolation.")
+
+    parser.add_argument("--trace",
+                        action="store_true",
+                        help="Use the tracing version of support.ll.")
+
+    parser.add_argument("-O0", action="store_true", help="Do no optimize.")
+    parser.add_argument("-O1", action="store_true", help="Use llc -O2.")
+    parser.add_argument("-O2",
+                        action="store_true",
+                        help="Use llc -O2 and opt -O2.")
+
+    parser.add_argument("--base", help="Load address to employ in lifting.")
+
+
+def run_translate(args, search_path, command_prefix):
+    main_path = path.abspath(__file__)
+    main_path = path.dirname(main_path)
+
+    command = [get_command("revng-pipeline", search_path)]
+    command.append("-P")
+    command.append(main_path + "/pipelines/translate.yml")
+
+    command.append("-P")
+    command.append(main_path + "/pipelines/isolate-translate.yml")
+
+    out_file = args.output if args.output else args.input[0] + ".translated"
+     
+    step_name = "EndRecompile"
+    if args.isolate:
+        step_name = step_name + "Isolated"
+    command = command + ["output:root:Translated",
+                         "--step",
+                         step_name,
+                         "-i",
+                         "Lift:input:" + args.input[0],
+                         "-o",
+                         step_name + ":output:" + out_file]
+
+    if not args.verbose:
+        command.append("--silent")
+
+    if args.O1:
+        command.append("--Compile-OptLevel=1")
+    elif args.O2:
+        command.append("--Compile-OptLevel=2")
+        command.append("-f")
+        command.append("O2")
+    else:
+        command.append("--Compile-OptLevel=0")
+
+    if args.base:
+        command.append("--base={}".format(args.base))
+
+    if args.trace:
+        command.append("--link-trace")
+
+    if args.showCommand:
+        print(" ".join(command))
+        return 0
+
+    return run(command, command_prefix)
+    
```

Comment:
You're leaving whitespaces at the end of line.
Please configure your editor to automatically purge those and double check all the patchset.
`git diff develop`, `/ $` to see all the occurrences.

Reply:
wait, i am confused, i added the newline there because in some other file where there was not you asked me to add it